packages/cozy-client/src/utils.js

```diff
@@ -1,19 +1,21 @@
+/**
+ * @typedef {Promise} CancelablePromise
+ * @property {Function} cancel - Cancel the promise
+ */
+
 /**
  * Wraps a promise so that it can be canceled
  *
  * Rejects with canceled: true as soon as cancel is called
  *
- * @typedef CancelablePromise {Promise}
- * @property {Function} [cancel] - Cancel the promise
- *
  * @param  {Promise} promise  - Promise
  * @returns {CancelablePromise} - Promise with .cancel method
  */
 const cancelable = promise => {
   let _reject
 
   /**
-   * @type {CancelablePromise}
+   * @type {Partial<CancelablePromise>}
```

Comment:
What is `Partial` ?

Reply:
https://www.typescriptlang.org/docs/handbook/utility-types.html

> Constructs a type with all properties of Type set to optional. This utility will return a type that represents all subsets of a given type.

Reply:
Thanks!